### ISSUE TITLE
ERROR: Unrecognized parameter: -F

### ISSUE THREAD

**Issue by blaiseli** [arq5x/bedtools2#370]
The documentation mentions the existence of a `-F` option for `bedtools intersect`: <http://bedtools.readthedocs.org/en/latest/content/tools/intersect.html>

However, when I try to use it (with `-F 1`), I obtain the following:

````
***** ERROR: Unrecognized parameter: -F *****
***** ERROR: Unrecognized parameter: 1 *****

Tool:    bedtools intersect (aka intersectBed)
Version: v2.23.0
Summary: Report overlaps between two feature files.

Usage:   bedtools intersect [OPTIONS] -a <bed/gff/vcf/bam> -b <bed/gff/vcf/bam>

	Note: -b may be followed with multiple databases and/or 
	wildcard (*) character(s). 
Options: 
	-ubam	Write uncompressed BAM output. Default writes compressed BAM.

	-bed	When using BAM input (-abam), write output as BED. The default
		is to write output in BAM when using -abam.

	-wa	Write the original entry in A for each overlap.

	-wb	Write the original entry in B for each overlap.
		- Useful for knowing _what_ A overlaps. Restricted by -f and -r.

	-loj	Perform a "left outer join". That is, for each feature in A
		report each overlap with B.  If no overlaps are found, 
		report a NULL feature for B.

	-wo	Write the original A and B entries plus the number of base
		pairs of overlap between the two features.
		- Overlaps restricted by -f and -r.
		  Only A features with overlap are reported.

	-wao	Write the original A and B entries plus the number of base
		pairs of overlap between the two features.
		- Overlapping features restricted by -f and -r.
		  However, A features w/o overlap are also reported
		  with a NULL B feature and overlap = 0.

	-u	Write the original A entry _once_ if _any_ overlaps found in B.
		- In other words, just report the fact >=1 hit was found.
		- Overlaps restricted by -f and -r.

	-c	For each entry in A, report the number of overlaps with B.
		- Reports 0 for A entries that have no overlap with B.
		- Overlaps restricted by -f and -r.

	-v	Only report those entries in A that have _no overlaps_ with B.
		- Similar to "grep -v" (an homage).

	-f	Minimum overlap required as a fraction of A.
		- Default is 1E-9 (i.e., 1bp).
		- FLOAT (e.g. 0.50)

	-r	Require that the fraction overlap be reciprocal for A and B.
		- In other words, if -f is 0.90 and -r is used, this requires
		  that B overlap 90% of A and A _also_ overlaps 90% of B.

	-s	Require same strandedness.  That is, only report hits in B
		that overlap A on the _same_ strand.
		- By default, overlaps are reported without respect to strand.

	-S	Require different strandedness.  That is, only report hits in B
		that overlap A on the _opposite_ strand.
		- By default, overlaps are reported without respect to strand.

	-split	Treat "split" BAM or BED12 entries as distinct BED intervals.

	-sorted	Use the "chromsweep" algorithm for sorted (-k1,1 -k2,2n) input.

	-g	Provide a genome file to enforce consistent chromosome sort order
		across input files. Only applies when used with -sorted option.

	-header	Print the header from the A file prior to results.

	-nobuf	Disable buffered output. Using this option will cause each line
		of output to be printed as it is generated, rather than saved
		in a buffer. This will make printing large output files 
		noticeably slower, but can be useful in conjunction with
		other software tools and scripts that need to process one
		line of bedtools output at a time.

	-names	When using multiple databases, provide an alias for each that
		will appear instead of a fileId when also printing the DB record.

	-filenames	When using multiple databases, show each complete filename
			instead of a fileId when also printing the DB record.

	-sortout	When using multiple databases, sort the output DB hits
			for each record.

	-iobuf	Follow with desired integer size of read buffer.
		Optional suffixes K/M/G supported.
		Note: currently has no effect with compressed files.

Notes: 
	(1) When a BAM file is used for the A file, the alignment is retained if overlaps exist,
	and exlcuded if an overlap cannot be found.  If multiple overlaps exist, they are not
	reported, as we are only testing for one or more overlaps.

````

This option would be useful if it existed, as `-a` and `-b` files do not play a symmetrical role.

**Comment by arq5x** [comment]
This option did not become available until version 2.25:. Sorry for the confusion: http://bedtools.readthedocs.org/en/latest/content/history.html#version-2-25-0-3-sept-2015

**Comment by blaiseli** [comment]
I had a quick look at <https://github.com/arq5x/bedtools2/releases> and didn't see mention of the -F option. I should have had a look at the link concerning version 2.25.
